src/www/ui/template/upload_permissions.html.twig

```diff
@@ -9,6 +9,10 @@
 
 {% block content %}
   
+  {% if message != "" %}
+  {{message}}
+  <br/><br/>
+  {% endif %} 
```

Comment:
```suggestion
```

The block already exists: https://github.com/fossology/fossology/blob/a3c93461b11aefc5a6b5d0ceb4a4a34ed8621cee/src/lib/php/UI/template/include/base.html.twig#L36-L56